### PR TITLE
Adds a subsection on reporting bugs in the websites managed by the R-Core team

### PR DESCRIPTION
**Pull request by razekmh**
At RSECon24 R dev day.

This resolves one part of #117

Changes made:
I have added a section 4.5.3 for the list of websites (R FAQ, Windows FAQ, Mac FAQ, R project web-site, CRAN website).

Not done:
Also mentioned in #117 is adding a new section for additional resources.

### CODE REVIEW

File: 04-reporting_bugs.Rmd [hunk 1]
```diff
@@ -74,6 +74,17 @@ Once you confirm a bug exists, you need to submit a bug report so that it gets f
...
+### Bug in a documentation website managed by the R-Core team {#docWebsRCoreBug}
+
+The R-Core team manages a list of website connected to the R project. If you noticed a bug or an issue which you would like to report, please [see posting guide](https://www.r-project.org/posting-guide.html#which_list) and then submit the issue report by sending an email to R-devel@R-project.org
+
+The list of websites managed by the R-Core team: 
+- [R FAQ](https://cran.r-project.org/doc/FAQ/R-FAQ.html)
+- [R for Windows FAQ](https://cran.r-project.org/bin/windows/base/rw-FAQ.html)
+- [R for macOS FAQ](https://cran.r-project.org/bin/macosx/RMacOSX-FAQ.html)
+- [The R Project website](https://www.r-project.org/)
+- [The Comprehensive R Archive Network](https://cran.r-project.org/)
```
Comment (cjrace) [inline 04-reporting_bugs.Rmd:86]: I found a few different emails on the sites, think an updated version of this with those included could look like:

```
The R-Core team also maintain a selection of websites that each have their own ways for reporting issues outside of Bugzilla. Have a read through their [posting guide](https://www.r-project.org/posting-guide.html#which_list) and then follow the instructions below to give feedback to the maintainers.

1. [R FAQ](https://CRAN.R-project.org/doc/manuals/R-FAQ.html) contains answers to some of the most frequently asked questions about R. Feedback on this site should be provided to [R-devel@R-project.org](mailto:R-devel@R-project.org).

2. [R for Windows FAQ](https://cran.r-project.org/bin/windows/base/rw-FAQ.html) contains answers specific to the Windows port of R. Comments on the FAQ site can be sent to [R-windows@R-project.org](mailto:R-windows@R-project.org).

3. [R for macOS FAQ](https://cran.r-project.org/bin/macosx/RMacOSX-FAQ.html) contains a user guide to the R.APP version of R, and information on using R on macOS, supplementing the main R manuals. To submit requests, ask questions or give feedback, subscribe to the [R-SIG-Mac mailing list](https://stat.ethz.ch/mailman/listinfo/r-sig-mac), the Special Interest Group on Mac ports of R, and then post your message to the mailing list by emailing [r-sig-mac@r-project.org](mailto:r-sig-mac@r-project.org).

4. Queries about [The Comprehensive R Archive Network website](https://www.r-project.org/) should be sent to [webmaster@r-project.org](mailto:webmaster@r-project.org).

5. Queries about the [CRAN website](https://cran.r-project.org/) should be sent to [cran-sysadmin@r-project.org](mailto:cran-sysadmin@r-project.org).
```

**Review by cjrace — COMMENTED** [review]
I accidentally duplicated some of this when picking up #117 at an R dev day. I did a scout through each of the websites and I think a few of them specific different ways to report feedback other than just through the main r-devel@ email. 

To save duplication I've deleted my branch on my fork, and have added these as suggestions incase they're helpful on top of what's already here.

I've raised a new PR to cover the rest of #117 in #186.

**Comment by hturner** [comment]
@razekmh thanks for pulling this list together.

@cjrace is correct that the contact address is different for each website and it would be helpful to have this documented all in one place.

Can you either submit an updated pull request, or [give us permission to make the updates](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)?

**Comment by hturner** [comment]
Thanks @razekmh for giving permission to edit, I have merged in @cjrace updates with a little editing so this can be merged in.

@all-contributors please add @razekmh for content and documentation.

**Pull request merged**